### PR TITLE
Update parallax example with a new parallax image window

### PR DESCRIPTION
**Pull request by aghassemi**
/to @cvializ

### CODE REVIEW

File: examples/article-parallax.amp.html [hunk 1]
```diff
@@ -6,7 +6,19 @@
   <link rel="canonical" href="amps.html">
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-fx-parallax" src="https://cdn.ampproject.org/v0/amp-fx-parallax-0.1.js"></script>
```
Comment (cvializ) [inline examples/article-parallax.amp.html]: Do you need `custom-element` if this extension does not register an `<amp-fx-parallax>` element?

File: examples/article-parallax.amp.html [hunk 2]
```diff
@@ -35,51 +47,55 @@
...
-  <script async custom-element="amp-fx-parallax" src="https://cdn.ampproject.org/v0/amp-fx-parallax-0.1.js"></script>
   <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
 </head>
 <body>
   <header>
+    <!-- Parallax Title -->
     <h1 amp-fx-parallax="1.7">
       <div class="vertically-center">
-        <span>Lorem Ipsum Dolor Sit Amet Consectetur Adipiscing<span>
+        <span amp-fx-parallax=1.7>Lorem Ipsum Dolor Sit Amet Consectetur Adipiscing<span>
```
Comment (cvializ) [inline examples/article-parallax.amp.html]: What are our standards on quotes vs. no quotes around attribute values?

File: examples/article-parallax.amp.html [hunk 3]
```diff
@@ -35,51 +47,55 @@
...
+      </p>
+      <p>
+        Mauris aliquet eleifend lacinia. Maecenas non finibus ante, et maximus urna. In non enim a lectus molestie tincidunt. Curabitur eu tortor odio. Aliquam bibendum elit risus, vitae rutrum ex placerat eget.
+      </p>
+      <p>
+        Etiam egestas velit ut purus dictum, vel placerat turpis convallis. Nullam metus urna, posuere ut elit eu, imperdiet consequat metus. Phasellus blandit at lorem ut mollis. Donec vel ligula at erat mattis aliquet eget quis nibh.
+      </p>
+      <!-- Parallax Image Window -->
+      <div class="parallax-window">
+        <amp-img amp-fx-parallax=1.2 height="100vh" layout="fixed-height" src="https://unsplash.it/1920/1280?image=1044"></amp-img>
```
Comment (cvializ) [inline examples/article-parallax.amp.html]: How does `layout="fixed-height"` work if the element is assigned a height value that varies with the height of the viewport? I suppose on mobile the viewport doesn't change size.

**Comment by aghassemi** [comment]
will be revisited

**Pull request closed without merging**